### PR TITLE
Adding new argument to 'Convert Json To String' keyword for pretty printing

### LINKED ISSUE #52: Pretty print when converting json to string
When using the 'Convert Json To String' keyword, the string is not really readable for bigger jsons.
This could easily be solved by using the indent parameter in the json.dumps method:
`return json.dumps(json_object, indent=4)`

I don't know if this can cause problems for other use cases than printing a json or dumping it into a file, but if yes, the pretty printing could be done via an additional parameter for the 'Convert Json To String' keyword.

### PR DESCRIPTION
**Pull request by oetzus**
#52

### CODE REVIEW

File: JSONLibrary/jsonlibrary.py [hunk 1]
```diff
@@ -202,18 +202,19 @@ def delete_object_from_json(self, json_object, json_path):
...
 
         Arguments:
             - json_object: json as a dictionary object.
+            - indent: indent level for pretty-printing, see indent argument of python's [https://docs.python.org/3/library/json.html#json.dump|json.dump()] for details
 
         Return new json_string
 
         Examples:
-        | ${json_str}=  |  Convert JSON To String | ${json_obj} |
+        | ${json_str}=  |  Convert JSON To String | ${json_obj} | ${4} |
```
Comment (elrandira) [inline JSONLibrary/jsonlibrary.py]: please add an example without indent

Comment (elrandira) [reply JSONLibrary/jsonlibrary.py]: add as well a test based on  test_convert_json_to_string

**Comment by oetzus** [comment]
@elrandira Can you have a look again?

**Comment by elrandira** [comment]
looks good to me

**Pull request merged**